src/board/v2/error.c

```diff
@@ -182,7 +182,7 @@ void HardFaultVector(void)
   static uint32_t *psp, *lr;
   asm("mrs %0, psp" : "=r"(psp) : : );
   asm("mov %0, lr" : "=r"(lr) : : );
-  
+
```

Comment:
Is there supposed to be a space here? My editor must have automatically removed when I was looking at this file...